src/protagonist/Engine.Tests/Ingest/Image/ImageServer/Clients/CantaloupeThumbsClientTests.cs

```diff
@@ -299,6 +311,106 @@ public async Task GenerateThumbnails_ReturnsThumbForSuccessfulResponse_AfterFirs
             "Landscape images - invalid sizes altered",
         },
     };
+    
+    [Fact]
+    public async Task GenerateThumbnails_UpdatesHandlerWithCookies()
+    {
+        // Arrange
+        var assetId = new AssetId(2, 1, nameof(GenerateThumbnails_ReturnsThumbForSuccessfulResponse));
```

Comment:
can be `var id = AssetIdGenerator.GetAssetId();` or `var id = AssetIdGenerator.GetAssetId(2, 1);` if cust / space are important.